app/controllers/users_controller.rb

```diff
@@ -42,17 +42,17 @@ def update
     if @user.update(permitted_attributes(@user))
       RssReaderFetchUserWorker.perform_async(@user.id) if @user.feed_url.present?
       notice = "Your profile was successfully updated."
-      if config_changed?
-        notice = "Your config has been updated. Refresh to see all changes."
-      end
```

Comment:
this should probably stay

Reply:
@citizen428 asked me to remove this piece of code. Once manual refresh is fixed.